core/crypto/src/signature.rs

```diff
@@ -701,6 +713,17 @@ impl<'de> serde::Deserialize<'de> for Signature {
     }
 }
 
+#[cfg(feature = "schemars")]
+impl schemars::JsonSchema for Signature {
+    fn schema_name() -> std::borrow::Cow<'static, str> {
+        "Signature".to_string().into()
+    }
+
+    fn json_schema(generator: &mut schemars::SchemaGenerator) -> schemars::Schema {
+        String::json_schema(generator)
+    }
+}
```

Comment:
I wonder why JsonSchema impl didn't work for SignedTransaction, but it seems to work here